tests/db/schemas/mssql.sql

```diff
@@ -69,6 +69,7 @@ CREATE TABLE runs (
 	status VARCHAR(9) COLLATE "SQL_Latin1_General_CP1_CI_AS",
 	start_time BIGINT,
 	end_time BIGINT,
+	delete_time BIGINT,
```

Comment:
Nit: I think we'll need to update these schemas to reflect the rename to `deleted_time`:

```suggestion
	deleted_time BIGINT,
```

Reply:
👌